Tests/common/CommonExtensions.swift

```diff
@@ -48,7 +48,7 @@ extension IterableAPI {
                            networkSession: @escaping @autoclosure () -> NetworkSessionProtocol = MockNetworkSession(),
                            notificationStateProvider: NotificationStateProviderProtocol = SystemNotificationStateProvider(),
                            inAppSynchronizer: InAppSynchronizerProtocol = MockInAppSynchronizer(),
-                           inAppDisplayer: InAppDisplayerProtocol = MockInAppDisplayer(),
+                           iterableMessageDisplayer: IterableMessageDisplayerProtocol = MockIterableMessageDisplayer(),
```

Comment:
I think `inAppDisplayer` is a better name if we're not planning to have generic messages for the time being.